Remove unnecessary `replaceall` dependency in favour of `replaceAll` (in `@graphql-tools/prisma-loader`)

GamerBene19 opened the pull request:
## Description

I've replaced all usages of `replaceall` with `replaceAll` (needs a cast to `string` since `property` is of type `any`) and removed the package `replaceall` from the appropriate `package.json`. Those changes should "fix" #4593. dependency

Related #4593.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)


## How Has This Been Tested?

I ran `yarn ts:check`, `yarn prettier:check` and `yarn test` all of which passed/completed w/o errors. However I'm unsure how I'd "properly" test this. I'm happy to test my changes if someone tells me how.

**Test Environment**:
- OS: Arch Linux
- `@graphql-tools/prisma-loader`: `7.2.5`
- NodeJS: `v17.9.0`

## Checklist:

- [x] I have followed the [CONTRIBUTING](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) doc and the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas  
  I don't think there are comments necessary - let me know if I should add some
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works  
  As mentioned above I'm not sure how I'd do that. Please advise.
- [x] New and existing unit tests and linter rules pass locally with my changes
  I haven't added new tests/linter rules.
- [ ] Any dependent changes have been merged and published in downstream modules
  I'm not quite sure what is meant by that, so I've left it unticked.

ardatan commented:
I think it is not supported by older versions of Node?
 Maybe we can replace it with `split` and `join` combination.

GamerBene19 commented:
> I think it is not supported by older versions of Node? Maybe we can replace it with `split` and `join` combination.

It's supported since NodeJS v15 [Mozilla Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll). So depending on what you define as "older versions of Node" the answer is either yes or no. 
Node v14 is in "Maintenance LTS" until end of April next year.

ardatan commented:
But we still support Node 14 so we cannot drop that support yet :)

GamerBene19 commented:
> But we still support Node 14 so we cannot drop that support yet :)

Sorry, did not know that - is this something that should be picked up again next year then?

ardatan commented:
We can basically use `.split` and `.join` combination which does the same thing;
```ts
str.split('FOO').join('BAR')
```

GamerBene19 commented:
> We can basically use `.split` and `.join` combination which does the same thing;
> 
> ```ts
> str.split('FOO').join('BAR')
> ```

Sorry for the late reply, I've missed the notification.
Should I change the code to use this approach and add a comment to replace it with `replaceAll` once Node v14 is no longer supported?

ardatan commented:
That makes sense! Thanks!

n1ru4l reviewed (changes requested):
Please add a changeset

GamerBene19 commented:
> That makes sense! Thanks!

Done ^^

GamerBene19 commented:
> Please add a changeset

Sorry, I don't understand what you need/want me to do - could you elaborate please?

ardatan commented:
We use `changesets` to manage releasing, changelog and so on.
https://github.com/ardatan/graphql-tools/pull/4594#issuecomment-1193183432
So in order to release this as a next patch version, you need to run `yarn changeset` to add a "changeset"

GamerBene19 commented:
> We use `changesets` to manage releasing, changelog and so on. [#4594 (comment)](https://github.com/ardatan/graphql-tools/pull/4594#issuecomment-1193183432) So in order to release this as a next patch version, you need to run `yarn changeset` to add a "changeset"

Thanks for the explanation! I've added a changeset. Let me know if there is anything else that I need to do :smile:

ardatan commented:
Awesome! Thanks!
